ray-operator/config/crd/patches/webhook_in_rayservice_rayservices.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
```

Comment:
Could you more about what this file is? Was it autogenerated?

Reply:
Autogenerated